FunctionWithReturn

```diff
@@ -0,0 +1,13 @@
+/*
+
+Create a function timesFive that accepts one argument, multiplies it by 5, and returns the new value.
+
+*/
+
+
+function timesFive(num){
+  return num*5;
+}
+
+let answer = timesFive(10);
+console.log(answer); //50
```

Comment:
Add EOF

Reply:
I don't know much about EOF. I will study about EOF in JS and I'll edit the file.